packages/core/scripts/util/ios.ts

```diff
@@ -18,8 +18,8 @@ export const exportSourcemapRNVariable =
   'export SOURCEMAP_FILE="$CONFIGURATION_BUILD_DIR/main.jsbundle.map";';
 
 export const EMBRACE_IMPORT_OBJECTIVEC = "#import <Embrace/Embrace.h>";
-export const EMBRACE_INIT_OBJECTIVEC =
-  "[[Embrace sharedInstance] startWithLaunchOptions:launchOptions framework:EMBAppFrameworkReactNative];";
+export const EMBRACE_INIT_OBJECTIVEC = `[[Embrace sharedInstance] startWithLaunchOptions:launchOptions
+                                           framework:EMBAppFrameworkReactNative];`;
```

Comment:
I'm not sure about this change, since it's injected into the customer's app delegate I think this will end up including extra whitespace